addons/pos_reprint/static/src/js/reprint.js

```diff
@@ -14,11 +16,140 @@ devices.ProxyDevice.include({
     },
 });
 
+screens.ReceiptScreenWidget.include({
```

Comment:
We might avoid this if we split the rendering of QWeb and DOM manipulation in ```point of sale```module